### ISSUE TITLE
[RI2 Ch4] Placeholder: Airship-based RI2 included in chapter04.md.

### ISSUE THREAD

**Issue by tomkivlin** [cntt-n/CNTT#2041]
To be replaced by more specific issues as needed.

**Comment by tomkivlin** [comment]
@pgoyal01 could you work with James to spawn some more useful and specific issues for this please?

**Comment by tomkivlin** [comment]
@pgoyal01 @jgu17 - are there any updates on this please?  Just wondering if we need to work out any dependencies that need issues raising?

**Comment by jgu17** [comment]
@tomkivlin yes, there are a couple of dependencies: acquire a pod in LAAS for CNTT Airship 2.0 installer (WIP); Airship 2.0 release (expect in 1Q 2021). 

I checked out the current chapter. It looks good but needs to be organized to allow to add the content for a second RI installer besides kuberef. That could be the first task once we are getting close to AS 2.0 availability?

**Comment by tomkivlin** [comment]
AS 2.0 available last week. @jgu17 are you able to do a brief PR for this in the next week please?  If not, we can move to backlog.

**Comment by jgu17** [comment]
@tomkivlin It was only the AS 2.0 beta. The production release is sometime in Q1 2021 (we may have a better idea of the exact date next week). We've identified the POD to use for AS 2.0 which should be ready for us by the end of this week or next week. We will put AS 2.0 production release (when it is released) on the new POD and create content for this chapter based the AS 2.0 deployment in the CNTT lab. The new POD is HPE server, so there will be some new development work expected to support the HPE hardware. Long story short, we won't make it next week.
